modules/cfe_testcase/src/cfe_test.c

```diff
@@ -41,6 +41,11 @@ CFE_FT_Global_t CFE_FT_Global;
  */
 void CFE_TestMain(void)
 {
+    /* Constant Table information used by all table tests */
+    CFE_FT_Global.TblName           = "TestTable";
+    CFE_FT_Global.RegisteredTblName = "CFE_TEST_APP.TestTable";
+    CFE_FT_Global.TblFilename       = "test_tbl.tbl";
```

Comment:
If this is just used by cfe_test_table.c it's probably more appropriate to initialize there.

Reply:
It's used by multiple functions across multiple source files. The alternative is to re-initialize in each one of them or initialize them in the first one but that introduces a test execution order dependency which feels too brittle.

Reply:
In that case I'm good as-is.

Reply:
Specifically 

```
TBLContentAccessTestSetup();
    TBLContentMangTestSetup();
    TBLInformationTestSetup();
TBLRegistrationTestSetup();
```

An alternative would be to create a new function say `TBLTests_Setup` that only contains this initialization.

Reply:
I actually prefer the idea of making a "Setup" function here that does this init, then it can be passed as the setup function as part of the UtTest_Add for any test that uses this table struct.  That being said, its OK as is, I wouldn't hold this up.

Reply:
I don't think doing it as part of the setup function would work out because there is already a setup function which is used by most but not **all** of the table tests to register a table. Also since CFE_FT_Global is already instantiated in cfe_test.c I think it would make the most sense to also perform the initialization of this data just the once either there or perhaps it could be moved into cfe_test_table.c.

Reply:
Opened an issue for future work https://github.com/nasa/cFE/issues/1797